usage.md

```diff
@@ -8,45 +8,43 @@ Installation will be required for capabilities such as [self-provisioning](doc/d
-3. ESX - while this can and will function against vCenter, the placeholder installation script doesn't handle distributed port groups at this time
-4. DHCP - the VCH currently requires there be DHCP on the external network (-e flag if not "VM Network")
+Replace the `<fields>` in the example with values specific to your environment - this will install VCH to the specified resource pool of ESXi or vCenter, and the container VMs will be created under that resource pool. Bug #539 is tracking the container VM creation issue.
 
+Here is one resource pool path sample: /ha-datacenter/host/localhost/Resources/test 
+-gen flag is to generate certificates and configure TLS
+-force flag is to remove an existing datastore folder or VM with the same name:
 
-Replace the `<fields>` in the example with values specific to your environment - this will install to the top-level resource pool of the host (specify -g to generate certificates and configure TLS). Add -f to remove an existing folder or VM with the same name:
 ```
-bin/install.sh -g -t '<user>:<password>@<target-host>' -i <datastore-name> <vch-name>
+vic-machine -target target-host -iStore <datastore-name> -name <vch-name> -user <user> -passwd <password> -pool <resource pool path in govc format> -gen
 ```
 This will, if successful, produce output similar to the following:
 ```
-# Generating certificate/key pair - private key in vch-name-key.pem
-# Logging into the target
-# Uploading ISOs
-[02-04-16 23:16:55] Uploading... OK
-[02-04-16 23:16:58] Uploading... OK
-# Creating vSwitch
-# Creating Portgroup
-# Creating the Virtual Container Host appliance
-# Adding network interfaces
-# Setting component configuration
-# Configuring TLS server
-# Powering on the Virtual Container Host
-# Setting network identities
-# Waiting for IP information
-#
-# SSH to appliance (default=root:password)
-# root@x.x.x.x
-#
-# Log server:
-# https://x.x.x.x:2378
-#
-# Connect to docker:
-# docker -H x.x.x.x:2376 --tls --tlscert='vch-name-cert.pem' --tlskey='vch-name-key.pem'
-DOCKER_OPTS="--tls --tlscert='vch-name-cert.pem' --tlskey='vch-name-key.pem'"
-DOCKER_HOST=x.x.x.x:2376
+INFO[2016-04-29T10:50:47-05:00] ### Installing VCH ####
+INFO[2016-04-29T10:50:47-05:00] Generating certificate/key pair - private key in ./vch-name-key.pem
+INFO[2016-04-29T10:50:48-05:00] Validating supplied configuration
+INFO[2016-04-29T10:50:55-05:00] Creating a Resource Pool
+INFO[2016-04-29T10:50:56-05:00] Creating VirtualSwitch
+INFO[2016-04-29T10:50:56-05:00] Creating Portgroup
+INFO[2016-04-29T10:50:56-05:00] Creating appliance on target
+INFO[2016-04-29T10:50:58-05:00] Uploading images for container
+INFO[2016-04-29T10:50:58-05:00] 	bootstrap.iso
+INFO[2016-04-29T10:50:58-05:00] 	appliance.iso
+INFO[2016-04-29T10:51:07-05:00] Waiting for IP information
+INFO[2016-04-29T10:51:25-05:00] Initialization of appliance successful
+INFO[2016-04-29T10:51:25-05:00] #
```

Comment:
There is no point in having the `#` in the log output if it's not at the start of the line. This was originally included in a fashion such that it was possible to pipe the output of the install to a shell and have it correctly set up to work with the newly created VCH.
If using the log formatter like this the # must be added to the formatter, or we should drop it completely.